metrics/web/lib/dashboard/presentation/view_models/build_result_metric_view_model.dart

```diff
@@ -12,20 +12,28 @@ class BuildResultMetricViewModel extends Equatable {
   /// A list of [BuildResultViewModel]s.
   final UnmodifiableListView<BuildResultViewModel> buildResults;
 
-  /// A number of [buildResults] elements to display.
-  final int numberOfBuildsToDisplay;
-
   /// A maximum [Duration] of a finished build within the [buildResults].
   final Duration maxBuildDuration;
 
+  /// A [DateTime] of this view model's first build.
```

Comment:
```suggestion
  /// A [DateTime] of the metric period start this view model represents.
```
Would it describe this field a bit clearer?